pyNN/neuron/standardmodels/electrodes.py

```diff
@@ -238,7 +239,8 @@ def __init__(self, **parameters):
     def _generate(self):
         ## Not efficient at all... Is there a way to have those vectors computed on the fly ?
         ## Otherwise should have a buffer mechanism
-        self.times = numpy.arange(self.start, self.stop, max(self.dt, simulator.state.dt))
+        temp_num_t = int(round((self.stop - self.start) / max(self.dt, simulator.state.dt)))
+        self.times = self.start + max(self.dt, simulator.state.dt * 1e-3) * numpy.arange(temp_num_t)
```

Comment:
I don't think the 1e-3 should be here

Reply:
My bad! Thanks for spotting that. Fixed now.